src/app/components/phone-field/index.html

```diff
@@ -15,9 +15,10 @@
   >
     <sky-demo-page-property
         propertyName="defaultCountry"
+        defaultValue="US"
```

Comment:
Technically this would be `us` in lower case. I did just push a change to my current changes that allows for capitalized values but we should probably change this just to be safe.

Reply:
Actually, I just made the change myself :)